Packs/GuardiCore/ReleaseNotes/2_0_8.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### GuardiCore v2
+- Updated the Docker image to: *demisto/python3:3.10.10.48392*.
+
+- Fixed an issue where the **fetch-incidents** command failed to fetch incidents.
```

Comment:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.10.48392*.
- Fixed an issue where the ***fetch-incidents*** command failed to fetch new incidents.
```

Reply:
There is an issue with the update-release-notes, already opened a ticket.